### PR TITLE
chore: adjust node engines

### PR DESCRIPTION
**Pull request by kendallstrautman**
## 🔗 Relevant links

- [Preview link](https://dev-portal-git-BRANCH_NAME-hashicorp.vercel.app/PATH_TO_VIEW) 🔎
- [Asana task](url) 🎟️

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
 		"start:packer": "cross-env DEV_IO=packer npm run start",
 		"start:sentinel": "cross-env DEV_IO=sentinel npm run start",
 		"start:vagrant": "cross-env DEV_IO=vagrant npm run start",
-		"start:vault": "cross-env DEV_IO=vault npm run start",
+		"start:vault": "cross-env DEV_IO=vaulfe npm run start",
```
Comment (kendallstrautman) [inline package.json:23]:
```suggestion
		"start:vault": "cross-env DEV_IO=vault npm run start",
```

Comment (kendallstrautman) [reply package.json:23]: oops

File: package.json [hunk 2]
```diff
@@ -43,7 +43,7 @@
 	},
 	"engines": {
 		"npm": ">=7.0.0",
-		"node": ">=14.0.0"
+		"node": ">=14.x <16.x"
```
Comment (kendallstrautman) [inline package.json:46]: I _think_ [this syntax](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#engines) will work?

Comment (BRKalow) [reply package.json:46]: yea! https://jubianchi.github.io/semver-check/#/%3E%3D14.x%20%3C16.x/16.13.0

**Review by BRKalow — APPROVED** [review]
ty ty

File: package.json (hunk 2 above)
Comment (BRKalow) [inline package.json:46]:
```suggestion
		"node": ">=14.x <=16.x"
```

File: package.json [hunk 3]
```diff
@@ -43,7 +43,7 @@
 	},
 	"engines": {
 		"npm": ">=7.0.0",
-		"node": ">=14.0.0"
+		"node": ">=14.0.0 <=16.18.1"
```
Comment (BRKalow) [inline package.json:46]: @kendallstrautman this will break if a new 16.x patch release is adopted 🤔

Comment (kendallstrautman) [reply package.json:46]: cool, i'll use the .x syntax instead. I wasn't sure if that's what was messing with the bundle analysis check

**Pull request merged**